clinica/iotools/converters/oasis_to_bids/oasis_to_bids.py

```diff
@@ -104,14 +104,16 @@ def convert_single_subject(subj_folder, dest_dir):
         if not path.isdir(bids_subj_folder):
             os.mkdir(bids_subj_folder)
 
-        session_folder = path.join(bids_subj_folder, "ses-M00")
+        session_folder = path.join(bids_subj_folder, "ses-M000")
         if not os.path.isdir(session_folder):
             os.mkdir(path.join(session_folder))
             os.mkdir(path.join(session_folder, "anat"))
 
         # In order do convert the Analyze format to Nifti the path to the .img file is required
         img_file_path = str(next(Path(t1_folder).glob("*.img")))
-        output_path = path.join(session_folder, "anat", bids_id + "_ses-M00_T1w.nii.gz")
+        output_path = path.join(
+            session_folder, "anat", bids_id + "_ses-M000_T1w.nii.gz"
```

Comment:
Again, `bids_id` here should be `participant_id` for better  understanding.

```suggestion
            session_folder, "anat", f"{participant_id}_ses-M000_T1w.nii.gz}"
```